code/modules/blob/blob.dm

```diff
@@ -4,16 +4,21 @@
 	icon = 'icons/mob/blob.dmi'
 	icon_state = "blob"
 	light_range = 3
+	light_color = "#b5ff5b"
 	desc = "Some blob creature thingy"
 	density = 1
 	opacity = 0
 	anchored = 1
 	mouse_opacity = 2
+	layer = 4
 
 	var/maxHealth = 30
 	var/health
+	var/regen_rate = 5
 	var/brute_resist = 4
+	var/laser_resist = 5	// Special resist for laser based weapons - Emitters or handheld energy weaponry. Damage is divided by this and THEN by fire_resist.
```

Comment:
flamethrowers are broken and worthless. They need a lot of fixing before they can be considered for use as weapons

Welding torches are tools, not weapons. They're not bad, but theres something grossly wrong if a hand welder isn't any more efficient than a giant laser wired to the station's power grid.

Have you tested these changes, and attempted to fight a blob with these methods?

I agree with the direction, lasers are overused and could do with some reduced effectiveness. But these values are far too much